.travis.yml

```diff
@@ -2,28 +2,32 @@ language: php
 
 sudo: false
 
-php: [5.3, 5.4, 5.5, 5.6, 7.0, hhvm]
+php: [5.4, 5.5, 5.6, 7.0, 7.1, hhvm]
 
 matrix:
   fast_finish: true
   include:
     - php: 5.3
+      dist: precise
+    - php: 7.1
       env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 5.6
+    - php: 7.1
```

Comment:
Is it needed to specify `-php 7.1` several times? You can combine multiple env vars into single line as well (above `COMPOSER_FLAGS` and `SYMFONY_DEPRECATIONS_HELPER` are combined into single line).

Reply:
Well, we want separate jobs for the lowest deps and the unstable deps (these cannot be combined together, as they are about different versions of deps).

and if you don't specify the PHP version for a job, Travis defaults to 5.5.

Reply:
I see. So that was on purpose. Thanks for explaining.